src/algorithms/DouglasRachford.jl

```diff
@@ -5,14 +5,16 @@
 # [1] Eckstein, Bertsekas "On the Douglas-Rachford Splitting Method and the Proximal Point Algorithm for Maximal Monotone Operators*", Mathematical Programming, vol. 55, no. 1, pp. 293-318 (1989).
 #
 
-struct DRSIterator{I <: Integer, R <: Real, T <: BlockArray{R}} <: ProximalAlgorithm{I, T}
+mutable struct DRSIterator{I <: Integer, R <: Real, T <: BlockArray{R}} <: ProximalAlgorithm{I, T}
     x::T
     f
     g
+    cost::R
```

Comment:
Forget about the cost for now, this we will add later. Let's not modify anything in the `initialize` and `iterate` methods in this PR, only visualization stuff.